src/www/pages/consentPortraitRight.html

```diff
@@ -0,0 +1,21 @@
+<!-- page for giving consent to portrait right -->
+<div ng-controller="portraitRightController">
+    <h1>Portrait right</h1>
+    <p>Portrait right is a privacy law in the Netherlands which is applicable to people that are in pictures. If at an
+        event
+        of H.S.A. Confluente a picture of you has been taken, then (most of the time) you get the rights over whether
+        that
+        picture can be used and where.</p>
+    <p>At activities we make lots of pictures and we want to hang those pictures up in the honors room as well as post
+        them
+        on the website (for members only). By giving consent on this page, you give consent to H.S.A. Confluente to hang
+        pictures of you in the honors room and post them on the website (for members only), without being asked if that
+        is
+        okay for every single picture. At any point in time you can still request pictures to be removed from any source
+        as
+        stated in the privacy policy.</p>
+    <p>Giving consent here does not mean that you give consent to H.S.A. Confluente to post pictures on social media.
+        For
+        every picture we post on social media, we ask the people on those pictures if that is okay</p>
```

Comment:
I feel like this page should be _slightly_ more formal. For instance, using 'ask for permission' instead of 'ask if that is okay'